enclose variable into quotation mark

jahongirman opened the pull request:
without quotation mark around $sshPublicKey, I am getting an error that says "Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'KeyData'. Specified method is not supported."

azurecla commented:
Hi __@alinamstanciu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alinast). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@dominicbetts - Please review this public repo PR and submit an update. Thanks

dominicbetts commented:
Hi @alinamstanciu, thanks for pointing this out. I've submitted an update to fix the Node.js API links.

jabdurahmonov committed:
enclose variable into quotation mark

without quotation mark around $sshPublicKey, I am getting an error that says "Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'KeyData'. Specified method is not supported."
